### ISSUE TITLE
Can't find package.json

### ISSUE THREAD

**Issue by charlisi** [AppsFlyerSDK/appsflyer-capacitor-plugin#108]
# Report

## Plugin Version
appsflyer-capacitor-plugin@^6.10.3

## On what Platform are you having the issue?
android

## What did you do?
`rush add -p "appsflyer-capacitor-plugin@^6.10.3`
`npx cap sync`
`npx cap build android`

## What did you expect to happen?
Dependencies installed   

## What happened instead?
Error message saying can't find your package.json

## Please provide any other relevant information.
The solution should be easy and self contained so it works on any kind of repo. In the android build.gradle file:
Why don't you do "$buildFile/../../package.json" instead of "$rootDir/../node_modules/appsflyer-capacitor-plugin/package.json" ? Then you don't need workarounds in project that integrates with your SDK.

**Comment by pazlavi** [comment]
The fix is scheduled for the upcoming release (probably 2nd half ofFebruary).
For now you can use this version:
```
npm install appsflyer-capacitor-plugin@6.12.1-alpha-monorepo
```

**Comment by pazlavi** [comment]
Fixed in 6.13.0
